### PR TITLE
Arjun P1 Polishes

### PR DESCRIPTION
**Pull request by apothuku**
|New|Old|
|---|---|
|image|image|

|New|Old|
|---|---|
|image|image|

|New|Old|
|---|---|
|image|image|

|New|Old|
|---|---|
|image|image|

|New|Old|
|---|---|
|image|image|

|New|Old|
|---|---|
|image|image|

|New|Old|
|---|---|
|image|image|

|New|Old|
|---|---|
|image|image|

|New|Old|
|---|---|
|image|image|

### CODE REVIEW

File: src/css/graph.css [hunk 1]
```diff
@@ -90,12 +90,27 @@ svg.scatter-plot {
   background-color: #00ed89;
 }
 
+.key-circle.red::before {
+  background-color: #CD5958;
+}
+
 .key-circle.green-with-opacity::before {
```
Comment (alexwgraves) [inline src/css/graph.css]: Should we get rid of the unused CSS?

Comment (alexwgraves) [reply src/css/graph.css]: Or alternatively use black and white for these rather than the color it displays as. Since that's how we do it in other parts of the code

File: src/js/classes/Map.js [hunk 2]
```diff
@@ -109,19 +109,7 @@ class ColorScaleLegend {
...
-      .append("text")
-      .attr(
-        "x",
-        this.labelOffsetX + (this.labels.length - 1) * this.sectionWidth
-      )
-      .attr("y", this.labelOffsetY)
-      .attr("class", legendTextClassName)
-      .attr(BUCKET_ATTRIBUTE, this.labels[this.labels.length - 1])
-      .text(toPercent(this.labels[this.labels.length - 1], 0));
+      .text((_, i) => i === 0 ? "" : toPercent(this.labels[i], 0, false));
```
Comment (alexwgraves) [inline src/js/classes/Map.js:112]: Why this change? Because the bar goes to zero again so the offset isn't needed?

Comment (apothuku) [reply src/js/classes/Map.js:112]: This was part of the design change in Figma, Rong removed the min and max labels.

Comment (alexwgraves) [reply src/js/classes/Map.js:112]: Ah okay. I mean I guess lmao

File: src/css/graph.css (hunk 1 above)
Comment (apothuku) [reply src/css/graph.css]: Done

**Review by alexwgraves — COMMENTED** [review]
Can you check the hex codes that are used to create the hatch patterns and make sure those are changed over to the new colors?

**Comment by alexwgraves** [comment]
Wait jk I coded that dynamically nice, way to go past me

File: src/css/visualization.css [hunk 3]
```diff
@@ -126,12 +126,16 @@
...
 }
 
-.bar-average-line.purple {
-  border-right: 1px dotted #c14ef8;
+.bar-average-line.white-race {
+  border-right: 1px dotted #AADFED;
+}
+
+.bar-average-line.white {
+  border-right: 1px dotted rgba(255, 255, 255, 0.2);
```
Comment (alexwgraves) [inline src/css/visualization.css]: The `.bar-average-line.white` one should be removed right?

File: src/js/classes/Table.js [hunk 4]
```diff
@@ -131,7 +131,7 @@ class BarGraphCell extends Cell {
     this.element.appendChild(bar);
     // add the vertical line denoting the average
     const averageLine = document.createElement("div");
-    averageLine.className = "bar-average-line green";
+    averageLine.className = "bar-average-line white";
```
Comment (alexwgraves) [inline src/js/classes/Table.js]: This should be `white-race`, right? That seems like what you've been using for the others

Comment (apothuku) [reply src/js/classes/Table.js]: Lol the racial color is so confusing! It's supposed to be referring from this line in figma, but it's actually more gray, so I'll call it that.
Screen Shot 2022-08-29 at 20 25 53

**Review by alexwgraves — APPROVED** [review]
Thanks!
